javascript/javascript2/week1/Js2Week1.js

```diff
@@ -0,0 +1,102 @@
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+const lengthArr = [];
+let min;
+let shortestWord;
+
+function findShortestWord() {
+
+    for (let i = 0; i < danishWords.length; i++) {
+        lengthItem = danishWords[i].length;
+        lengthArr.push(lengthItem);
+        lengthArr.sort((a, b) => a - b)
+        min = lengthArr[0];
+        if (danishWords[i].length == min) {
+
+            shortestWord = danishWords[i];
+        }
+
+    }
+    return shortestWord;
+}
+
+shortestWord = findShortestWord();
+console.log(shortestWord);
+
+
+
+
+/*Find and count the Danish letters
+Find the individual number and the total number of Danish letters in a string.
+
+const danishString = "Jeg har en blå bil";
+notThisFunctionName(danishString); // returns {total: 1, å: 1}
+
+const danishString2 = "Blå grød med røde bær";
+notThisFunctionName(danishString2); // returns {total: 4, æ: 1, ø: 2, å: 1}*/
+
+const danishLetters = ['å', 'ø', 'æ',]
+const danishString = "Jeg har en blå bil";
+const danishString2 = "Blå grød med røde bær";
+
+function findDanishLetters(string) {
+    const resultObj = {};
+    let sum = 0;
+    string = string.split('');
```

Comment:
It is good practice not to mutate the parameter. Instead make a new variable. Also, string is not a good variable name, instead choose something that depicts what it is, like 'danishLetterArray'

Reply:
thank you, will follow your advice